app/views/trains/_form.html.erb

```diff
@@ -16,6 +16,11 @@
     <%= form.collection_select(:railway_station_id, RailwayStation.all, :id, :title) %>
   </div>
 
+  <div class="field">
+    <%= form.label :cars, 'Car' %>
+    <%= form.collection_select(:car_id, Car.all, :id, :number) %>
```

Comment:
I have the opportunity to select three cars?

Reply:
no. separately.

Reply:
remove